components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m

```diff
@@ -44,9 +44,6 @@ - (void)setUp {
   MDCSemanticColorScheme *colorScheme =
       [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
 
```

Comment:
Shouldn't this be replaced with a call to the theming extension?

Reply:
MDCTextInputControllerOutlinedTextArea doesn't have a theming extension. Only MDCTextInputControllerOutlined and MDCTextInputControllerFilled have theming extensions. The approach I'm taking in this PR is sort of similar to https://github.com/material-components/material-components-ios/pull/9196

Reply:
Thanks. Would you please leave a TODO at each location with a link to an Issue that requests the MDCTextInputControllerOutlinedTextArea theming extension?

Reply:
I think the TODOs left for #9196 are sufficient. Where the typography (and now color theming) code used to be it now says `// TODO: Theme the MDCTextInputController with a theming extension (to be added in #9199)`

Reply:
Ah, sorry I didn't see those. Looks good!